atlasdb-backup/src/main/java/com/palantir/atlasdb/backup/ExternalBackupPersister.java

```diff
@@ -43,9 +48,12 @@ public class ExternalBackupPersister implements BackupPersister {
     private static final String FAST_FORWARD_TIMESTAMP_FILE_NAME = "fast-forward.timestamp";
 
     private final Function<AtlasService, Path> pathFactory;
+    private final Supplier<ObjectMapper> legacyObjectMapper;
 
     public ExternalBackupPersister(Function<AtlasService, Path> pathFactory) {
         this.pathFactory = pathFactory;
+        this.legacyObjectMapper = Suppliers.memoize(
+                () -> OBJECT_MAPPER.copy().setPropertyNamingStrategy(PropertyNamingStrategies.KEBAB_CASE));
```

Comment:
is there any reason we can't just have a second static?

Reply:
I guess we could - we most likely won't need to initialize this, but doing so can't be _that_ expensive, and it'd make the code a little simpler.